docker/README.md

```diff
@@ -0,0 +1,14 @@
+# Aerogear Jenkins openshift docker image
+
+## Why another image?
+
+OpenShift Jenkins docker images use `source to image` aproach for customizing jenkins plugins and configuration. 
+We need to use it in order to make any change in jenkins configuration that would be persisted in image.
```

Comment:
Can we say this 
We make use of the source to image so that any changes made to the jenkins configuration would be persisted